module1/homework/lengthSort/sort.cpp

```diff
@@ -0,0 +1,14 @@
+#include "sort.hpp"
+
+#include <algorithm>
+
+bool compare(const std::string& first_string, const std::string& second_string) {
+    return first_string.size() < second_string.size();
+}
```

Comment:
What about the lexicographical part when sizes are the same? I know that our tests suck because they should detect it.

Reply:
We sort lexicographically input_list at the beginning of lengthSort function. (10 line)
And after this is sorting depeding on size.

Reply:
As we talked on Discord this solution with two sorts was not optimal. I changed a bit compare function, could you check this now?